graylog2-rest-client/src/main/java/org/graylog2/restclient/models/dashboards/widgets/SearchResultChartWidget.java

```diff
@@ -23,32 +23,22 @@
 
 import java.util.Map;
 
-public class SearchResultChartWidget extends DashboardWidget {
-
-    private static final int DEFAULT_WIDTH = 2;
-    private static final int DEFAULT_HEIGHT = 1;
-
-    private final String streamId;
-    private final String interval;
+public class SearchResultChartWidget extends ChartWidget {
 
     public SearchResultChartWidget(Dashboard dashboard, String query, TimeRange timerange, String description, String streamId, String interval) {
         this(dashboard, null, description, streamId, 0, query, timerange, null, interval);
     }
 
     public SearchResultChartWidget(Dashboard dashboard, String id, String description, String streamId, int cacheTime, String query, TimeRange timerange, String creatorUserId, String interval) {
-        super(Type.SEARCH_RESULT_CHART, id, description, cacheTime, dashboard, creatorUserId, query, timerange);
-
-        this.streamId = streamId;
-        this.interval = interval;
+        super(Type.SEARCH_RESULT_CHART, id, description, cacheTime, dashboard, creatorUserId, query, timerange, streamId, interval);
     }
 
     @Override
     public Map<String, Object> getConfig() {
         Map<String, Object> config = Maps.newHashMap();
         config.putAll(getTimerange().getQueryParams());
+        config.putAll(super.getConfig());
```

Comment:
Shouldn't this be the first addition to the map so that the query parameters and the local config can overwrite settings?